README.md

```diff
@@ -44,7 +44,7 @@ branch.
 | `release/0.5` | [`draft-ietf-ppm-dap-04`][dap-04] | Yes | Unmaintained as of June 24, 2024 |
 | `release/0.6` | [`draft-ietf-ppm-dap-07`][dap-07] | Yes, [with errata](#draft-ietf-ppm-dap-07-errata) | Unmaintained as of June 24, 2024 |
 | `release/0.7` | [`draft-ietf-ppm-dap-09`][dap-09] | Yes, [with errata](#draft-ietf-ppm-dap-09-errata) | Supported |
-| `main` | [`draft-ietf-ppm-dap-13`][dap-13] | Not yet (under development) | Supported |
+| `main` | [`draft-ietf-ppm-dap-14`][dap-14] | Not yet (under development) | Supported |
```

Comment:
If we're not aware of any remaining discrepancies from the draft, then we should just say "Yes". If we find any bugs, we'll either fix them or document errata.
```suggestion
| `main` | [`draft-ietf-ppm-dap-14`][dap-14] | Yes | Supported |
```